src/web-ui/xdn/routes.js

```diff
@@ -0,0 +1,98 @@
+import { Router } from "@xdn/core/router";
+import { CACHE_ASSETS, CACHE_PAGES } from "./cache";
+
+const DIST_APP = 'dist';
+const DIST_XDN = 'dist-xdn';
+
+const router = new Router();
+
+// static prerendering
+router.prerender([
+  // home
+  '/',
+]);
+
+// xdn static files
+router.get('/service-worker.js', ({ serviceWorker, cache }) => {
+  cache(CACHE_ASSETS);
+  serviceWorker(`${DIST_XDN}/service-worker.js`);
+});
+router.get('/main.js', ({ serveStatic, cache }) => {
+  cache(CACHE_ASSETS);
+  serveStatic(`${DIST_XDN}/browser.js`);
+});
+
+// aws services
```

Comment:
These are Retail Demo Store services. Calling them AWS services is misleading.

Reply:
Renamed